docroot/modules/custom/foia_file/foia_file.drush.inc

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @file
+ * FOIA Api module drush commands.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function foia_file_drush_command() {
+  $items = [];
+
+  $items['file-entity-updater'] = [
```

Comment:
I think drush commands are typically named in the form of actions, so this would be `file-entity-update`, rather than `updater`

Reply:
👍 I'll replace all instances of `updater` with `update`